Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginManager.swift

```diff
@@ -0,0 +1,113 @@
+import UIKit
+
+public final class SimplifiedLoginManager {
+    public enum SimplifiedLoginError: Error {
+        case noLoggedInSessionInSharedKeychain
+    }
+    
+    var keychainSessionStorage: KeychainSessionStorage?
+    let client: Client
+    var user: User?
+    
+    // Properties for building SimplifiedLoginViewController
+    let env: ClientConfiguration.Environment
+    let withMFA: MFAType?
+    let loginHint: String?
+    let extraScopeValues: Set<String>
+    let completion: LoginResultHandler
+    var withSSO: Bool = true
+    
+    @available(iOS, obsoleted: 13, message: "This function should not be used in iOS version 13 and above")
+    public init(accessGroup: String,
+                client: Client,
+                env: ClientConfiguration.Environment, // TODO: Currently used to decide language.
```

Comment:
Can we avoid env parameter? Client already contains ClientConfiguration, right?

Reply:
I will remove it completely within the context of this ticket https://github.schibsted.io/spt-identity/t/issues/1816

Reply:
Great! 😊